book.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'item'
+
+class Book < Item
+  def initialize(publisher, cover_stat)
```

Comment:
Good Job implementing the Book class. I see you add achieved as an argument in the super method  `super(publish_date, archived: archived)` which is a good idea. 

I think, since the archived is a parameter, it should be added in the initialize method such that the value is passed into the archive in the super method which will, in turn, update the parent class 
e.g `def initialize(publisher, cover_stat, archived)`
         ` super(publish_date, archived: archived)
` would be okay